File: api/src/main/java/org/example/api/dto/VideoDto.java
package org.example.api.dto;

import lombok.Data;

@Data
public class VideoDto {
    private Integer id;
    private String title;       // 标题
    private String author;      // 作者
    private String coverUrl;    // 封面
    private String videoUrl;    // 视频地址（http://xxx.mp4）
    private String duration;    // 时长
    private String level;       // 等级
}

File: healthy/app/src/main/java/com/example/myapplication/data/model/VideoDto.kt
package com.example.myapplication.data.model

import com.google.gson.annotations.SerializedName

/**
 * 对应后端的 VideoDto
 */
data class VideoDto(
    val id: Int,
    val title: String,       // 标题
    val author: String?,     // 作者 (对应后端的 author)
    val coverUrl: String?,   // 封面
    val videoUrl: String,    // 视频地址
    val duration: Int,       // 时长
    val level: String?,       // 等级 (对应后端的 level)
    val preparePose: String?,
    val actionProcess: String?,
    val breathRhythm: String?,
    val attention: String?
)